opentech/static_src/src/app/src/containers/GroupByRoundDetailView.js

```diff
@@ -1,18 +1,60 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux'
 
 import DetailView from '@components/DetailView';
 import ByRoundListing from '@containers/ByRoundListing';
+import {
+    getRoundsFetching,
+    getRoundsErrored,
+} from '@selectors/rounds';
+import {
+    getByGivenStatusesLoading,
+    getByGivenStatusesError,
+    getCurrentRoundSubmissions,
+    getCurrentSubmissionID,
+} from '@selectors/submissions';
 
-export default class GroupByRoundDetailView extends React.Component {
+class GroupByRoundDetailView extends React.Component {
     static propTypes = {
         submissionStatuses: PropTypes.arrayOf(PropTypes.string),
+        submissions: PropTypes.arrayOf(PropTypes.object),
+        submissionID: PropTypes.number,
+        isLoading: PropTypes.bool,
+        isErrored: PropTypes.bool,
     };
 
     render() {
         const listing = <ByRoundListing submissionStatuses={this.props.submissionStatuses} />;
+        const { isLoading, isErrored, submissions, submissionID } = this.props;
+        const isEmpty = submissions.length === 0;
+        const activeSubmision = !!submissionID;
+
         return (
-            <DetailView listing={listing} />
+            <DetailView
+                isEmpty={isEmpty}
+                listing={listing}
+                isLoading={isLoading}
+                showSubmision={activeSubmision}
+                error={isErrored ? 'Fetching failed.' : undefined}
```

Comment:
Can we not grab an error message from the store? 

Equally - pass in `isErrored` to be explicit.

so we can do:
```
    isErrored={ isErrored },
    errorMessage= { errorMessageFromStore || 'Fetching failed.' }
```

Reply:
Yeah, I prefer this. I guess we should now follow suit in `ByRoundListing`? https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/static_src/src/app/src/containers/ByRoundListing.js#L75

Reply:
Yes, lets be consistent.